README.md

```diff
@@ -99,7 +99,7 @@ Imagine that you take the code from this question, save it to a file called `sus
 
 What would be the output? Explain your answer.
 
-<!-- Replace this comment with your answer -->
+still there's an x in here, so it wouldn't work. but if replaced with howMuchILikeSushi it would evaluate to "sushi is tasty"
```

Comment:
Actually, as surprising as it might seem, this code will still work. The reason is that because it's in an `if...else`, and the first condition (`howMuchILikeSushi > 0`) is true, the line that would evaluate `x` never actually gets run. As a result, the code prints out "sushi is tasty".